files/.github/workflows/ci.yml

```diff
@@ -18,11 +18,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      <% if (pnpm) {%>- uses: wyvox/action-setup-pnpm@v2
-      <% } else { %>- name: Install Node
```

Comment:
Decided to remove the custom `name` value, as both actions provide a default one.
Makes this file easier to maintain as well, but do let me know if a custom name is preferred, and I'll revert.

Reply:
name always bothered me anyway, so I'm good with this :partying_face: